Vagrantfile

```
@@ -45,9 +45,9 @@ cd /fossology
 
 DEBIAN_FRONTEND=noninteractive ./utils/fo-installdeps -y
 
-sudo make clean
-make
-sudo make install
+sudo cmake -DCMAKE_BUILD_TYPE=MinSizeRel -S. -B./build -G Ninja 
```

Would recommend optimized builds for Vargrant
```suggestion
sudo cmake -DCMAKE_BUILD_TYPE=Release -S. -B./build -G Ninja 
```